lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -50,6 +51,12 @@ def do_execute(machine, command, options={})
           end
         }
 
+        if options[:metainfo]
```

Comment:
We want the following discrete states:
 - silent
 - verbose, with vagrant-provided error handling (for things like `dnf`, etc
 - verbose, with muted vagrant error handling (for Origin Bash steps that have their own stacktrace, etc)